ocaml/typing/ctype.ml

```diff
@@ -5847,6 +5872,8 @@ let rec build_subtype env (visited : transient_expr list)
       else (t, Unchanged)
   | Tunivar _ | Tpackage _ ->
       (t, Unchanged)
+  (* FIXME jbachurski: Can Tapp occur here, in subtype construction? *)
+  | Tapp _ -> assert false
```

Comment:
This can occur. I think that only the `Tapp(Tconstr Unapplied, ...)` case can result in building a useful subtype.

Reply:
Should fix in a later PR, but will expand comment.